ocfweb/docs/docs/staff/procedures/new-host.md

```diff
@@ -29,9 +32,11 @@ Unfortunately, these steps tend to change a lot as our infrastructure evolves.
 
 ### Step 1.1. Add the LDAP entry
 
-On supernova, `kinit $USER/admin ldap-add-host <hostname> <ip>`. If setting up
-a desktop, also do `kinit $USER/admin ldapvi cn=<hostname>` and set the `type`
-attribute to `desktop`. If doing a staff VM, set it to `staffvm` instead.
+On supernova, `kinit $USER/admin ldap-add-host <hostname> <ip-last-octet>`.
```

Comment:
This is not really an issue with this PR but somewhere we should explain our IP scheme (using the same last octet), which explains the arguments to ldap add host